src/http/Message.h

```diff
@@ -115,6 +115,9 @@ class Message : public RefCountable
     // Parser-NG transitional parsing of mime headers
     bool parseHeader(Http1::Parser &); // TODO move this function to the parser
 
+    /// descendants should configure the interpreter as needed
```

Comment:
s/descendants should configure/configures/ to simplify; this is a pure method so it is clear that descendants must override it